wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Open editor screen with blogging prompt

Opens editor screen a with blogging prompt content from feature onboarding and reminder notification.

Fixes #16121 

To test:
1 - Onboarding 
- Open Debug settings and enable `blogging_prompts_notification_remote_field`
- Open app logged in and with a site selected
- Click on the blogging prompts onboarding notification OR in the "Learn more" button
- Blogging prompts onboarding dialog should be seen
- Tap on "Try it Now" button
- Editor screen should be opened with a pullquote block

2 - Reminder notification
- Open Debug settings and enable `blogging_prompts_notification_remote_field`
- On `My Site`, tap on `Menu`
- Tap on `Site Settings`
- Tap on `Reminders and prompts` and select a date and time for the reminder
- Change the device date to something after the selected date/time for the reminder
- A notification to answer the prompt should appear
- Tap on the notification and the editor screen should be opened with a pullquote block

3 - My Site card
- Open Debug settings and enable `blogging_prompts_remote_field`
- On `My Site`, tap on `Home`
- Scroll down until you see the prompts card (with the text `Cast the movie of your life`)
- Tap on `"Answer prompt"` and the editor screen should be opened with a pullquote block

4 - Compact card
- Open Debug settings and enable `blogging_prompts_remote_field`
- On `My Site`, tap on the FAB. You should see the option to answer prompt.
- Tap on `"Answer prompt"` and the editor screen should be opened with a pullquote block

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually testing both scenarios

3. What automated tests I added (or what prevented me from doing so)
Updated `BloggingPromptsOnboardingViewModelTest`

PR submission checklist:
